metadata-ingestion/src/datahub/emitter/rest_emitter.py

```diff
@@ -129,6 +125,19 @@ def test_connection(self) -> None:
         response = self._session.get(f"{self._gms_server}/config")
         response.raise_for_status()
         config: dict = response.json()
+        if ":9002" in self._gms_server:
+            logger.warning(
```

Comment:
Not necessarily - going forward, datahub-frontend can serve as a proxy to GMS. People can point to datahub-frontend instead to minimize the access points they need to maintain.